src/test/java/com/gs/crdtools/GeneratorTest.java

```diff
@@ -36,14 +36,14 @@ void testGenerate() throws IOException {
 
         var input = Path.of(runFiles.rlocation("__main__/src/test/resources/minimal-crd.yaml"));
         var parsedCrd = Generator.parseCrds(List.of(input));
-        var sourceCodeFromSpecs = Generator.generate(parsedCrd);
+        var sourceCodeFromSpecs = new CustomGenerationTest.Result(Generator.generate(parsedCrd));
```

Comment:
If we move the Result definition to a separate file this will solve itself, but otherwise I think this is an excellent example of where it makes sense to use a static import and get rid of the surrounding class